# Sajjad Toghiani (PhD)  <br />
### *Research Geneticist*  <br />

<!--
[Email](mailto:sajjad.toghiani@usda.gov) / 
[LinkedIn](https://www.linkedin.com/in/sajjad-toghiani-7bab9552/) / 
[GitHub](https://github.com/stghn/) / 
[Researchgate](https://www.researchgate.net/profile/Sajjad-Toghiani/) / 
[GoogleScholar](https://scholar.google.com/citations?user=H9PqUOEAAAAJ&hl=en)/
-->

[![Gmail](https://img.shields.io/badge/Gmail-EA4335?logo=Gmail&logoColor=white&style=for-the-badge)](mailto:stoghiani@gmail.com)
[![Outlook](https://img.shields.io/badge/Outlook-0078D4?logo=MicrosoftOutlook&logoColor=white&style=for-the-badge)](mailto:sajjad.toghiani@usda.gov)
[![Linkedin](https://img.shields.io/badge/linkedin-0A66C2?logo=linkedin&logoColor=white&style=for-the-badge)](https://www.linkedin.com/in/sajjad-toghiani-7bab9552/)
[![Researchgate](https://img.shields.io/badge/ResearchGate-00CCBB?logo=ResearchGate&logoColor=white&style=for-the-badge)](https://www.researchgate.net/profile/Sajjad-Toghiani/)
[![Google Scholar](https://img.shields.io/badge/GoogleScholar-4285F4?logo=GoogleScholar&logoColor=white&style=for-the-badge)](https://scholar.google.com/citations?user=H9PqUOEAAAAJ&hl=en)
[![Github](https://img.shields.io/badge/GitHub-181717?logo=GitHub&logoColor=white&style=for-the-badge)](https://github.com/stghn/)


## Profile
<p align="justify">
Sajjad is an Research Geneticist (Animals) at <a href="https://www.ars.usda.gov/people-locations/person?person-id=55652" target="_blank">USDA in Animal Genomics and Improvement Laboratory</a> with a background in animal breeding and genetics. His areas of interest are focused on the application of genomic information in livestock including genomic selection, genome-wide association studies, genotype imputation, and population genomics. His goals are to conduct research using the cutting edge of genomic research in animal breeding to develop decision-making tools and improved algorithms to enhance animal health and welfare, environmental sustainability, climate adaptability, and and productivity cost-effectively for producers. 
</p>

## Education
**PhD   University of Georgia (Athens, GA, USA)**   &emsp; &emsp; &emsp;  2015-2018 <br />
        *College of Agricultural and Environmental Sciences* <br />
        *Department of Animal and Dairy Science*  <br />
         Majored in **Animal Breeding and Genetics** <br />

**MS   University of Georgia (Athens, GA, USA)**   &emsp; &emsp; &emsp;   2013-2015  <br />
       *College of Agricultural and Environmental Sciences*  <br />
		   *Department of Animal and Dairy Science*     <br />
		    Majored in **Animal Breeding and Genetics**  <br />

**MS    Guilan University (Rasht, IRAN)**    &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; 2005-2007 <br />
        *College of Agricultural Sciences*  <br />
        *Department of Animal Science*  <br />
         Majored in **Animal Breeding and Genetics** <br />

**BS     Shiraz University (Shiraz, IRAN)**   &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; 2001-2005 <br />
        *College of Agricultural Sciences*   <br />
        *Department of Animal Science*  <br />
         Majored in **Animal Science**  <br />

## Professional Experience
**USDA-ARS-AGIL -- Beltsville, MD, USA**   <br />
*Research Geneticist (Animals)*   <br />
January 2022 -- Present   <br />

**USDA-ARS-AGIL -- Beltsville, MD, USA**   <br />
*Animal Scientist*   <br />
March 2020 -- January 2022   <br />

**USDA-ARS-LRRL -- Miles City, MT, USA**  <br />
*Postdoctoral Research Fellow as Geneticist*  <br />
October 2018 -- March 2020  <br />

## Research Experience
**University of Georgia** , *Graduate Research Assistant (PhD)*  <br />
January 2015 -- August 2018  <br />
**PhD Dissertation:** *Implementing genomic selection method based on using Fst prioritized SNPs* <br />

**University of Georgia** , *Graduate Research Assistant (M.S.)*   <br />
January 2013 -- December 2015  <br />
**Master Thesis:** *Multi-generational imputation of SNP genotypes and accuracy of genomic selection* <br />

**Iowa State University**, *Laboratory Research Assistant*  <br />
September 2009 -- January 2010 <br />

**University of Guilan**, *Graduate Research Assistant (M.S.)* <br />
September 2005 -- September 2007   <br />
**Master Thesis:** *Estimation of genetic parameters for yield, fertility, and type traits in Iranian Holstein* <br />


## Publications
### Journal Publications

- **Toghiani, S.**, & VanRaden, P. (2021). National Index Correlations and Actual vs. Expected Use of Foreign Sires. Interbull Bulletin, (56), 52-59.
- Sumreddee, P., Hay, E. H., **Toghiani, S.**, Roberts, A., Aggrey, S.E., & Rekaya, R. (2021). Grid search approach to discriminate between old and recent inbreeding using phenotypic, pedigree and genomic information. BMC genomics, 22(1), 1-17.
- Sumreddee, P., **Toghiani, S.**, Hay, E. H., Roberts, A., Aggrey, S.E., and Rekaya, R. (2020). Runs of homozygosity and analysis of inbreeding depression. Journal of Animal Science, 98(12), skaa361.
- **Toghiani, S**., Hay, E., Roberts, A., and Rekaya, R. (2020). Impact of cold stress on birth and weaning weight in a composite beef cattle breed. Livestock Science, 104053.
- **Toghiani, S**., Hay, E., Fragomeni, B., Rekaya, R., and Roberts, A.J. (2020). Genotype by environment interaction in response to cold stress in a composite beef cattle breed. Animal-The International Journal of Animal Biosciences, 1-12.
- Aggrey, S. E., **Toghiani, S**., Chang, L. Y., and Rekaya, R. (2020). Improving accuracy of genomic prediction using a selected small set of prioritized SNP markers. Proceedings, 68th Annual Poultry Breeders' Round Table Conference.
- Chang, L. Y., **Toghiani, S**., Hay, E. H., Aggrey, S. E., and Rekaya, R. (2019). A Weighted Genomic Relationship Matrix Based on Fixation Index (FST) Prioritized SNPs for Genomic Selection. Genes, 10(11),922.
- Chang, L. Y., **Toghiani, S**., Aggrey, S. E., and Rekaya, R. (2019). Increasing accuracy of genomic selection in presence of high-density marker panels through the prioritization of relevant polymorphisms. BMC genetics, 20(1), 21.
- Sumreddee, P., **Toghiani, S**., Hay, E. H., Roberts, A., Aggrey, S.E., and Rekaya, R. (2018). Inbreeding depression in Line1 Hereford cattle population using pedigree and genomic information. Journal of Animal Science, 97(1), 1-18.
- Rekaya, R., **Toghiani, S**., Sumreddee, P., Ling, A., and Aggrey, S. (2018). Multivariate genome wide association for continuous and discrete responses using multivariate Bernoulli prior. Journal of Animal Science, 96(suppl_3), 126-126.
- Chang, L. Y., **Toghiani, S.**, Ling, A., Aggrey, S. E., and Rekaya, R. (2018). High density marker panels, SNPs prioritizing and accuracy of genomic selection. BMC genetics, 19(1), 4.
- **Toghiani, S.**, Hay, E., Sumreddee, P., Geary, T. W., Rekaya, R., and Roberts, A. J. (2017). Genomic prediction of continuous and binary fertility traits of females in a composite beef cattle breed. Journal of animal science, 95(11), 4787-4795.
- **Toghiani, S.**, Chang, L. Y., Ling, A., Aggrey, S. E., and Rekaya, R. (2017). Genomic differentiation as a tool for single nucleotide polymorphism prioritization for Genome wide association and phenotype prediction in livestock. Livestock Science, 205, 24-30.
- Chang, L. Y., **Toghiani, S.**, Ling, A., Hay, E. H., Aggrey, S. E., and Rekaya, R. (2017). Analysis of multiple binary responses using a threshold model. Journal of Agricultural, Biological and Environmental Statistics, 22(4), 640-651.
- Sumreddee, P., **Toghiani, S.**, Aggrey, S. E., and Rekaya, R. (2017).Joint genome-wide association analysis of continuous and discrete traits. Journal of Animal Science, 95(suppl_4), 104-105.
- Chang, L. Y., **Toghiani, S.**, Aggrey, S. E., and Rekaya, R. (2017). Increasing accuracy of genomic selection in presence of high-density marker panels through the prioritization of relevant polymorphisms. Journal of Animal Science, 95(suppl_4), 91-92.
- **Toghiani, S.**, Chang, L. Y., Aggrey, S. E., and Rekaya, R. (2017). A hybrid of prioritized SNP and polygenetic effect method for implementation of genomic selection. Journal of Animal Science, 95(supplement4), 92-92.
- Chang, L. Y., **Toghiani, S**., Aggrey, S. E., and Rekaya, R. (2016). High density marker panels, SNPs prioritizing and accuracy of genomic selection. Journal of Animal Science, 94(suppl_5), 141-142.
- **Toghiani, S**., Chang, L. Y., Aggrey, S. E., and Rekaya, R. (2016). SNP filtering using Fst and implications for genome wide association and phenotype prediction. Journal of Animal Science, 94(suppl_5), 143-143.
- **Toghiani, S**., Aggrey, S. E., and Rekaya, R. (2016). Multi-generational imputation of single nucleotide polymorphism marker genotypes and accuracy of genomic selection. Animal-The International Journal of Animal Biosciences, 10(7), 1077-1085.
- Foroughi, M., Najafi, P., **Toghiani, S**., and Honarjoo, N. (2014). Using Ceratophyllum demersum for Treatment of Saline Municipal Wastewater. Journal of Residuals Science & Technology, 11(2).
- Foroughi, M., Najafi, P., **Toghiani, S**., Toghiani, A., and Honarjoo, N. (2013). Nitrogen Removals by Ceratophyllum Demersum from Wastewater. Journal of Residuals Science & Technology, 10(2).
- **Toghiani, S**. (2012). Genetic relationships between production traits and reproductive performance in Holstein dairy cows. Archives Animal Breeding, 55(5), 458-468.
- Foroughi, M., and **Toghiani, S**. (2012). Phosphorus removal from aquaculture wastewater and latex by Ceratophyllum demersum. African Journal of Biotechnology, 11(7), 1639-1643.
- Foroughi, M., Najafi, P., and **Toghiani, S**. (2011). Trace elements removal from wastewater by Ceratophyllum demersum. Journal of Applied Sciences and Environmental Management, 15(1). 
- **Toghiani, S**. (2011). Genetic parameters and correlations among linear type traits in the first lactation of Holstein Dairy cows. African Journal of Biotechnology, 10(9), 1507-1510.
- **Toghiani Pozveh, S**., Shadparvar, A. A., Shahrbabak, M. M., and Taromsari, M. D. (2009). Genetic analysis of reproduction traits and their relationship with conformation traits in Holstein cows. Livestock Science, 125(1), 84-87.

### Conference Proceedings (Poster)

- Sumreddee, P., **Toghiani, S.**, Hay, E., Aggrey, S. E., and Rekaya, R. PSXII-29 Partitioning of Inbreeding Depression using Pedigree and Genomic Approaches, Journal of Animal Science, Volume 98, Issue Supplement_4, November 2020, Pages 247--248.
- **Toghiani, S**., Hay, E., Fragomeni, B., Rekaya, R., and Roberts, A. J. Genotype by environment interaction in response to cold stress in a composite beef cattle breed. Plant and Animal Genome XXVIII Conference, San Diego, CA, January 11 - 15, 2020.
- Sumreddee, P., **Toghiani, S**., Hay, E., Roberts, A., Aggrey, S. E., and Rekaya, R. Runs of homozygosity and analysis of inbreeding depression. Plant and Animal Genome XXVIII Conference, San Diego, CA, January 11 - 15, 2020.
- Chang, LY., **S. Toghiani**, S. Aggrey, and R. Rekaya. A weighted genomic relationship matrix based on Fst prioritized SNPs for genomic selection. American Society of Animal Science meeting, 2019, Austin, Texas.
- Sumreddee, P., **S. Toghiani**, S. Aggrey, and R. Rekaya. Joint genome-wide association analysis of continuous and discrete traits. American Society of Animal Science meeting, 2017, Baltimore, Maryland.
- **Toghiani, S**., L. Chang, and R. Rekaya. Genome wide association in presence of high-density marker panels and genotyped causal variants. The Allied Genetics Conference, 2016, Orlando, Florida.
- **Toghiani, S**., and R. Rekaya. Multi-generational imputation of SNP genotypes and accuracy of genomic selection. Joint Annual Meeting 2015, Orlando, Florida.


### Conference Proceedings (Oral)

- **Toghiani, S.,** P. VanRaden. National Index Correlations and Expected vs. Actual Use of Foreign Sires. Interbull Virtual Conference, 2021, Leeuwarden, The Netherlands.
- **Toghiani, S**., L. Chang, Hay, E. H., Roberts, S. Aggrey, and R. Rekaya. A hybrid model for genomic selection using prioritized SNPs based on Fst scores in the presence of non-genotyped animals. American Society of Animal Science meeting, 2019, Austin, Texas.
- **Toghiani, S**., L. Chang, S. Aggrey, and R. Rekaya. A hybrid of prioritized SNP and polygenetic effect method for implementation of genomic selection. American Society of Animal Science meeting, 2017, Baltimore, Maryland.
- Chang, LY., **S. Toghiani**, S. Aggrey, and R. Rekaya. Increasing accuracy of genomic selection in presence of high-density marker panels through the prioritization of relevant polymorphisms. American Society of Animal Science meeting, 2017, Baltimore, Maryland.
- **Toghiani, S**., L. Chang, S. Aggrey, R. Rekaya. SNP filtering using Fst and implications for genome wide association and phenotype prediction. Joint Annual Meeting 2016, Salt Lake City, Utah.
- Chang, LY., **S. Toghiani**, S. Aggrey, and R. Rekaya. High density marker panels, SNPs prioritizing and accuracy of genomic selection. Joint Annual Meeting 2016, Salt Lake City, Utah.


### Book Chapters
- **Toghiani, S**. (2012). Quantitative Genetic Application in the Selection Process for Livestock Production. In Livestock Production. InTech.

## Editorial Activities
Frontiers in Genetics -- Section Livestock Genomics: <a href="https://www.frontiersin.org/research-topics/24065/reducing-the-environmental-footprint-of-livestock-through-genomic-selection" target="_blank">Reducing the Environmental Footprint of Livestock Through Genomic Selection</a>.


## Honors and Awards
- **PhD graduate assistantship** awarded by USDA-ARS Livestock and Range Research Laboratory (Miles city, MT)
- **Master graduate assistantship** awarded by department of Animal and Dairy science (University of Georgia)

## Computer Skills

**Programming:** Fortran, SAS, R, Python, Unix shell scripting, Git and Github  <br />
**Applications:** MS office (Word, Excel, PowerPoint, Outlook)  <br />
**Analysis Tools:** **Plink** *(GWAS analysis software)*, **Gensel** *(Bayesian method for GS/GWAS analysis)*, **Blupf90 family** *(Genetic/Genomic evaluation)*, **QMsim** *(QTL and marker simulator for livestock populations)*, **FImpute** *(Genotype imputation tool)*,**Bioconductor** *(Statistical and graphical methods for analyzing genomic data)*, **Admixture** *(A tool to estimate individual ancestry from SNP genotype datasets)*


#### Resume: [[HTML](https://stghn.github.io/)] [[DOC](Curriculum_Vitae.docx)]

